src/main/java/frc/robot/Robot.java

```diff
@@ -24,8 +24,8 @@ public class Robot extends TimedRobot {
 
   private RobotContainer m_robotContainer;     
   
-  private final Climber climber = new Climber(Constants.Climber.MotorID);
-  private final Elevator elevator = new Elevator(Constants.Elevator.Lift.MotorID);
+  private final Climber climber = new Climber();
```

Comment:
No, we should not be having the subassemblies read off constants like this; they should be passed in. This style will allow us to decouple the subassembly code from any particular components which will allow us to duplicate subassemblies as needed (look at how the swerve modules work) and even build a subassembly library in the future.